service.kn.switchtimer/addtimer.py

```diff
@@ -0,0 +1,17 @@
+import xbmc
+import handler
+
+if __name__ ==  '__main__':
+
+    handler.notifyLog('Parameter handler called: add Timer')
+    try:
+        args = {'channel':xbmc.getInfoLabel('ListItem.ChannelName'),
```

Comment:
a better way would be to use sys.listitem instead of xbmc.getInfoLabel('ListItem.xxx').
(because stuff runs async and because listitem.xxx infolabel only works inside media windows)